src/main/scala/com/campudus/tableaux/helper/JsonUtils.scala

```diff
@@ -7,6 +7,7 @@ import com.campudus.tableaux.database.model.TableauxModel.{ColumnId, Ordering}
 import com.campudus.tableaux.{ArgumentCheck, FailArg, InvalidJsonException, OkArg}
 import com.typesafe.scalalogging.LazyLogging
 import org.vertx.scala.core.json.{Json, JsonArray, JsonObject}
+import com.campudus.tableaux.{InvalidJsonException,WrongJsonTypeException, TableauxConfig}
```

Comment:
[Minor] Missing space between `InvalidJsonException,` and `WrongJsonTypeException`